### PR TITLE
[SPARK-25676][FOLLOWUP][BUILD] Fix Scala 2.12 build error

### PR DESCRIPTION
**Pull request by dongjoon-hyun**
## What changes were proposed in this pull request?

This PR fixes the Scala-2.12 build.

## How was this patch tested?

Manual build with Scala-2.12 profile.

### CODE REVIEW

**Comment by dongjoon-hyun** [comment]
@dbtsai . The PR is ready. Could you review this?

**Comment by dbtsai** [comment]
LGTM. Thanks.

**Comment by dbtsai** [comment]
Merged into master as the compilation finished. Thanks!

**Comment by dongjoon-hyun** [comment]
Thank you, @dbtsai !

**Comment by dongjoon-hyun** [comment]
The failures are irrelevant to this PR because this PR only updates benchmark code.

File: sql/core/src/test/scala/org/apache/spark/sql/execution/benchmark/WideTableBenchmark.scala [hunk 1]
```diff
@@ -42,7 +43,7 @@ object WideTableBenchmark extends SqlBasedBenchmark {
       Seq("10", "100", "1024", "2048", "4096", "8192", "65536").foreach { n =>
         benchmark.addCase(s"split threshold $n", numIters = 5) { iter =>
           withSQLConf(SQLConf.CODEGEN_METHOD_SPLIT_THRESHOLD.key -> n) {
-            df.selectExpr(columns: _*).foreach(identity(_))
+            df.selectExpr(columns: _*).foreach((x => x): Row => Unit)
```
[the remark this answers is not in the document]
Comment (dongjoon-hyun) [reply sql/core/src/test/scala/org/apache/spark/sql/execution/benchmark/WideTableBenchmark.scala]: Sure. I'll try to make another follow-ups with the benchmark result.

**Pull request closed without merging**